support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYouAusMap.jsx

```diff
@@ -0,0 +1,59 @@
+// @flow
+import React from 'react';
+import { css } from '@emotion/core';
+import { space } from '@guardian/src-foundations';
+import { LinkButton } from '@guardian/src-button';
+import { SvgArrowRightStraight } from '@guardian/src-icons';
+import ActionContainer from './components/ActionContainer';
+import ActionHeader from './components/ActionHeader';
+import ActionBody from './components/ActionBody';
+import SvgAusMap from './components/SvgAusMap';
+import { OPHAN_COMPONENT_ID_AUS_MAP } from './utils/ophan';
+import { trackComponentClick } from 'helpers/tracking/behaviour';
+
+const buttonContainer = css`
+  margin-top: ${space[6]}px;
+`;
+
+const AUS_MAP_URL = 'https://support.theguardian.com/aus-2020-map';
```

Comment:
let's keep the tracking query string from the old thankyou page

Reply:
good spot! 👍